packages/domain-search/src/components/domain-suggestion/featured.scss

```diff
@@ -9,7 +10,7 @@
 .domain-suggestion-featured__body,
 .domain-suggestion-featured__content,
 .domain-suggestion-featured__content--small {
-	height: 100%;
+	height: 100% !important;
```

Comment:
It's not a blocker, but is there any way to avoid the `!important` operator?

Reply:
We can use inline styles or more specific selectors, but they aren't any better than important specifiers.